diff --git a/build.gradle b/build.gradle
index 88c6268..b19dd82 100644
--- a/build.gradle
+++ b/build.gradle
@@ -56,6 +56,9 @@ tasks.named('processResources', ProcessResources).configure {
 }
 
 tasks.named('jar', Jar).configure {
+    from(rootProject.projectDir) {
+        include 'LICENSE', 'NOTICE'
+    }
     manifest {
         attributes([
             'Specification-Title': mod_name,
diff --git a/src/main/resources/META-INF/mods.toml b/src/main/resources/META-INF/mods.toml
index d9f90fa..ba105d1 100644
--- a/src/main/resources/META-INF/mods.toml
+++ b/src/main/resources/META-INF/mods.toml
@@ -18,7 +18,7 @@ Original project: https://github.com/Provismet/Cursed-Spawners
 [[dependencies.${mod_id}]]
 modId="forge"
 mandatory=true
-versionRange="[47.2,)"
+versionRange="[47.4.16,)"
 ordering="NONE"
 side="BOTH"
 
diff --git a/src/main/resources/assets/cursed-spawners/sounds.json b/src/main/resources/assets/cursed-spawners/sounds.json
index 7ce5a65..0d64956 100644
--- a/src/main/resources/assets/cursed-spawners/sounds.json
+++ b/src/main/resources/assets/cursed-spawners/sounds.json
@@ -15,7 +15,10 @@
   },
   "block.spawner.knockback": {
     "sounds": [
-      "minecraft:random/explode"
+      {
+        "name": "minecraft:entity.generic.explode",
+        "type": "event"
+      }
     ],
     "subtitle": "subtitles.block.spawner.knockback"
   },
